src/Symfony/Component/Workflow/WorkflowInterface.php

```diff
@@ -19,6 +19,8 @@
  * Describes a workflow instance.
  *
  * @author Amrouche Hamza <hamza.simperfit@gmail.com>
+ *
+ * @method Transition|null getEnabledTransition(object $subject, string $name)
```

Comment:
This should be reverted. We can do this change in 7.1.

Reply:
Alright, I'll add it if it isn't done during upmerge :)